### DISCUSSION TITLE
kiedy-wszystko-zaczyna-sie-walic-cloudflare-email-routing/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#118] [General]
# kiedy-wszystko-zaczyna-sie-walic-cloudflare-email-routing/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/kiedy-wszystko-zaczyna-sie-walic-cloudflare-email-routing/

**Comment by spexpl** [comment]
Czy dobrze rozumiem, iż Gmailify,com, pozwala synchronizować outlooka <> gmail, ale bez wykorzystania POP3/IMAP?

A jak sobie radzi z etykietami gmail i wysyłaniem maili?

**Reply by idarek** [reply]
Nie używam Gmailify.com tylko Gmailify od Google (to są dwie różne usługi). Google Gmailify synchronizuje Outlook > Gmail (nie obustronnie) ale do końca nie wiem jak technicznie jest to rozwiązane. Wygląda jak IMAP ale czy jest to IMAP to nie wiem.

**Reply by spexpl** [reply]
A znasz jakiś sposób na synchronizację potem w drugą stronę? 

I czy synchronizacja (ta od google) synchronizuje usunięcia? W sensie jak usuniesz coś w Outlook (co już jest w gmail) to czy w Gmail też potem zostanie usunięte?

**Reply by idarek** [reply]
Co do pierwszego, nie.

Co do drugiego, jeżeli coś jest w Outlook i pokazane na Gmail za pomocą Gmailify od Google, to jak usuniesz na Google, usunie się na Outlook. Czasami jest tak, że jak usuniesz w Outlook to też usunie w Gmail, ale nie znam szczegółów jak to działa.

Poczytaj tutaj: https://support.google.com/mail/answer/6304825
